tests/plugins/neg/divideZero/scalac-plugin.xml

```diff
@@ -0,0 +1,4 @@
+<plugin>
```

Comment:
We should check that sbt's support for compiler plugin still works and adapt it in sbt-dotty if it doesn't: http://www.scala-sbt.org/1.x/docs/Compiler-Plugins.html We can do that by adding scripted tests in https://github.com/lampepfl/dotty/tree/master/sbt-dotty/sbt-test/sbt-dotty